validator.go

```diff
@@ -87,6 +87,9 @@ func newValidatorImpl(domains []string, usersFile string,
 	}
 
 	validator := func(email string) (valid bool) {
+		if allowAll {
+			valid = true
```

Comment:
We can just return here, no need to do anything more
```suggestion
			return true
```